frontend/src/components/InfoPanel.jsx

```diff
@@ -149,8 +151,8 @@ class InfoPanel extends React.PureComponent {
       `Set counterfactual first restrictions date to ${this.state.dateFirstRestrictionsCounterfactual}`
     );
     await Promise.all([
-      this.loadStatisticsCounterfactual(),
       this.loadAllowedDates(),
+      this.loadStatisticsCounterfactual(),
```

Comment:
Switching the order inside `Promise.all` won't affect the order that they finish in - it will still start them both at ~the same time.

If we want to run one then the other I think we need:

```
await this.loadAllowedDates();
await this.loadStatisticsCounterfactual();
```

Basically the tradeoff is between speed (running several commands in parallel) and ordering (knowing that one operation will have finished before the next one starts).

Reply:
This is now implemented in 9436143.